crates/syntax/src/ast/edit_in_place.rs

```diff
@@ -487,6 +483,26 @@ fn normalize_ws_between_braces(node: &SyntaxNode) -> Option<()> {
     Some(())
 }
 
+pub trait Indent: AstNode + Clone + Sized {
+    fn indent_level(&self) -> IndentLevel {
+        IndentLevel::from_node(self.syntax())
+    }
+    fn indent(&self, level: IndentLevel) -> &Self {
```

Comment:
```suggestion
    fn indent(&self, level: IndentLevel) {
```

guess we shouldn' return anything here, to make this clear that this is a modification.

Reply:
Lots of `edit::AstNodeEdit` usages are chained, I thought to support that. But right, it obscures API change.